device/core/src/device_client.ts

```diff
@@ -9,12 +9,14 @@ const debug = dbg('azure-iot-device:DeviceClient');
 
 import { AuthenticationProvider, RetryOperation, ConnectionString, results, Callback, ErrorCallback, callbackToPromise } from 'azure-iot-common';
 import { InternalClient, DeviceTransport } from './internal_client';
-import { BlobUploadClient } from './blob_upload';
+import { BlobUploadClient, UploadParams, BlobUploadResult2 } from './blob_upload';
 import { SharedAccessSignatureAuthenticationProvider } from './sas_authentication_provider';
 import { X509AuthenticationProvider } from './x509_authentication_provider';
 import { SharedAccessKeyAuthenticationProvider } from './sak_authentication_provider';
 import { DeviceMethodRequest, DeviceMethodResponse } from './device_method';
-import { DeviceClientOptions } from './interfaces';
+import { DeviceClientOptions, BlobUploadCommonResponseStub } from './interfaces';
+
+
 
```

Comment:
nit: watch out for unnecessary blank lines

Reply:
noted.